readme.md

```diff
@@ -16,6 +16,43 @@ The submission will be reviewed and added to the Fate SRD. This usually happens
 
 If you are comfortable with editing files and git/Github, you may [create a pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request). It will be reviewed and added to the site.
 
+There is a Github check configured to ensure the CSV file is in the expected order. This check is implemented a small
+Javascript script that runs under [Nodejs](https://nodejs.org/) version 20.x. (The code ill probably run under older versions, too.)
```

Comment:
Do we want to formally declare node 20 with an .npmrc file in the repo?

Reply:
Another way to do that is via the "[engines](https://docs.npmjs.com/cli/v10/configuring-npm/package-json#engines)" key in package.json. TBH, I thought I had done that already.

Do you have a preference for which technique to use?  I don't know if there is a functional difference; I think putting it in the package.json file makes it more obvious, developers will more likely read package.json than a hidden file in the repo.

Reply:
I tested the code with node 18 and node 19, both threw an error on the usage of `Array.prototype.toSorted()`, so I have added in an `"engines": { "node": "20.x" }` stanza to package.json, which causes node 18 and node 19 fail the npm install (or npm ci) steps.